### PR TITLE
Bugfix for ERROR Attempted method: security_groups_provider_updated and

### PR DESCRIPTION
**Pull request by VTabolin**
VimFaultException: The resource ID is in use

### CODE REVIEW

File: vmware_dvs/tests/unit/utils/test_dvs_util.py [hunk 1]
```diff
@@ -750,7 +750,8 @@ def test_creates_network_map_from_conf(self, *args):
             vmware_conf.vsphere_login,
             vmware_conf.vsphere_password,
             vmware_conf.api_retry_count,
-            vmware_conf.task_poll_interval)
+            vmware_conf.task_poll_interval,
+            pool_size=100)
```
Comment (ekosareva) [inline vmware_dvs/tests/unit/utils/test_dvs_util.py]: Let's move pool_size constant it in config file(/vmware_dvs/common/config.py,  vmware_opts) and use smth like:
pool_size = vmware_conf.connections_pool_size

File: vmware_dvs/utils/dvs_util.py [hunk 2]
```diff
@@ -534,12 +552,15 @@ def blocked(self, value):
...
+    # It needs to increase connection pool for multiple
+    # requests
     connection = api.VMwareAPISession(
         config.vsphere_hostname,
         config.vsphere_login,
         config.vsphere_password,
         config.api_retry_count,
-        config.task_poll_interval)
+        config.task_poll_interval,
+        pool_size=100)
```
Comment (ekosareva) [inline vmware_dvs/utils/dvs_util.py]: Let's move pool_size constant it in config file(/vmware_dvs/common/config.py,  vmware_opts)

File: vmware_dvs/tests/unit/utils/test_dvs_util.py (hunk 1 above)
Comment (VTabolin) [inline vmware_dvs/tests/unit/utils/test_dvs_util.py]: good idea but I had no enough time on 200-nodes lab to do so

File: vmware_dvs/utils/dvs_util.py (hunk 2 above)
Comment (VTabolin) [inline vmware_dvs/utils/dvs_util.py]: done at next commit

**Pull request merged**